sirepo/pkcli/admin.py

```diff
@@ -64,6 +64,28 @@ def create_examples():
                     if example.models.simulation.name not in names:
                         _create_example(example)
 
+# TODO(e-carlin): more than uid (ex email)
+def delete_user(uid):
```

Comment:
@robnagler [you mentioned](https://github.com/radiasoft/sirepo/issues/3620#issuecomment-849799813) wanting a generic concept for identifying a user. Can you elaborate a bit on what you are thinking?